SalesforceCommerceCloud/pwa-kit
# Replace dropdown for quantity with chakra number input

**GUS**: (https://gus.my.salesforce.com/a07AH000000L4nbYAC)

 The description the bug in the ticket in not reproducible as 10 items has been set as the maximum, I decided not to change this as there is an inventory ticket coming behind this that will address it - This PR fixes the bug and changes the quantity dropdown to an appropriate number stepper for desktop and mobile

@jennmills I had to make some design choices on mobile breakpoint as the new stepper is bigger that the former one, so I moved price above it - let me know if you have an alternate approach 

## How to test-drive this PR
- Check out and run
- Check that the bug regarding a quantity of 10 being added to the cart bugged out the quantity on cart
- Check that the mobile stepper is displayed on mobile breakpoints
